addons/google_calendar/models/calendar.py

```diff
@@ -88,12 +88,6 @@ def _odoo_attendee_commands(self, google_event):
         attendee_commands = []
         partner_commands = []
         google_attendees = google_event.attendees or []
-        if google_event.organizer and google_event.organizer.get('self', False):
```

Comment:
I think, although it should be double-checked, that when the organizer is the only attendee of a Google Calendar Event (created in google calendar), the attendees list is not initialized by Google. So there is no "attendees" key in Google's response. If that's indeed the case, then leave this part and change if cond to `if len(google_attendees) == 0 and google_event.organizer and google_event.organizer.get('self', False).`

Reply:
You're right!
I updated the commit